Makefile

```diff
@@ -68,6 +76,12 @@ docker-test:
 .PHONY: supported-platforms
 supported-platforms: $(LINUX_BINARY) $(DARWIN_BINARY)
 
+$(WINDOWS_BINARY): $(SOURCES)
+	@mkdir -p ./bin/windows-amd64
+	TARGET_GOOS=windows GOARCH=amd64 ./scripts/build_binary.sh ./bin/windows-amd64
+	mv ./bin/windows-amd64/ecs-cli ./bin/windows-amd64/ecs-cli.exe
```

Comment:
The .exe tells windows that its a program that is supposed to executed. If you don't have .exe, then windows will sometimes try to open the file. For example, when I was testing, if I tried to name the binary just `ecs-cli` and add it to my Windows path, then when I entered `ecs-cli` in PowerShell it would open a dialog prompting me to choose a program to open the file (it was trying to read the binary file instead of execute it). Whereas, if you name the binary `ecs-cli.exe`, then `ecs-cli` will be the command that tells Windows to run the executable file `ecs-cli.exe`.

Reply:
Windows uses file extensions to define what happens to a file when you try to open it. `.exe` one of the executable file extensions and generally the one we want to use (there are others, but for different purposes).